doc/api/esm.md

```diff
@@ -501,6 +501,19 @@ a nested conditional does not have any mapping it will continue checking
 the remaining conditions of the parent condition. In this way nested
 conditions behave analogously to nested JavaScript `if` statements.
 
+#### Resolving custom conditions
+
+When running Node.js, custom comma-separated conditions can be set with the
+`--conditions` or `-m` flag:
```

Comment:
```suggestion
`--conditions` or `-u` flag:
```

But I would maybe just mention the `--conditions` variant. Savvy users will find the short-hand.